Cactooz/StarWarsWiki
# Scroll bug

Changes:
- Always goes to top when routing to non browse pages

How to test:
- Scroll down on any browseview and click on a card to see that the scroll is on top.
- Scroll down on any browseview and click on "Profile" to see that the scroll is on top.
- Scroll down on any browseview and click on "Star Wars Wiki logo of green Yoda with red Christmas hat" to see that the scroll is on top.

Todo:
- Fix so that the remembered scroll position on browseviews do not get reset if you return from a page with no/small scroll
